Develop

blinchik3000 opened the pull request:
только основная часть

blinchik3000 committed:
second commit
check develop branch

blinchik3000 committed:
third commit
add dependencies to pom
new tests
new Classes and Objects

blinchik3000 committed:
forth commit
allure

mkouznetsov-git commented on src/main/java/orders/OrderGenerator.java:
```diff
@@ -0,0 +1,40 @@
+public class OrderGenerator {
+    private static final String[] colors = new String[] { "BLACK", "GREY" };
+    private static final int randomColor = new Random().nextInt(colors.length);
+//    private static YearMonth = n
```
Всё что лишнее -- лучше удалять, старайся не коммитить ненужные комментарии в финальной версии на проверку.

mkouznetsov-git commented on src/test/java/courier/CourierCreateTest.java:
```diff
@@ -0,0 +1,63 @@
+
+import static org.hamcrest.Matchers.equalTo;
+
+public class CourierCreateTest {
```
Не хватает теста: "если создать пользователя с логином, который уже есть, возвращается ошибка". Этот кейс отличается от полного дубликата, который тоже нужен по заданию.

mkouznetsov-git commented on src/test/java/orders/OrderListTest.java:
```diff
@@ -0,0 +1,29 @@
+import static org.hamcrest.Matchers.notNullValue;
+
+public class OrderListTest {
+    private Order order;
```
Эта переменная реально не используется в тестах, похоже, она лишняя.

blinchik3000 committed:
fifth commit
delete unnecessary comments
delete unnecessary variable
new test for double client with the same login only
